__tests__/cli.ts

```diff
@@ -0,0 +1,76 @@
+import fs from 'fs';
+import path from 'path';
+import util from 'util';
+import childProcess from 'child_process';
+
+const mkdir = util.promisify(fs.mkdir);
+const rmdir = util.promisify(fs.rmdir);
+const writeFile = util.promisify(fs.writeFile);
+
+function exec(
+  command: string,
+  options: { cwd: string },
+): Promise<{ exitCode: number | null; stdout: string }> {
+  return new Promise((resolve) => {
+    const process = childProcess.exec(command, options, (_error, stdout) => {
+      resolve({ exitCode: process.exitCode, stdout });
+    });
+  });
+}
```

Comment:
This _could_ move out into a `utils` module or something if we want to.  Might help for some reuse?  There's already #19 also trying to "promisify" `childProcess.exec` so maybe we want to hone in on a common solution?

Reply:
100%

Reply:
Not to self for tomorrow:

![image](https://user-images.githubusercontent.com/23029903/105362267-ce1acf00-5c4e-11eb-8e60-1ee63f0b6412.png)